RELEASE.md

```diff
@@ -60,52 +60,57 @@ Following Requirements needs to be met prior to final RC Cut:
 
 Release branches are typically cut from the branch [`viable/strict`](https://github.com/pytorch/pytorch/tree/viable/strict) as to ensure that tests are passing on the release branch.
 
-There's a convenience script to create release branches from current `viable/strict` (from root `pytorch/pytorch`):
+There's a convenience script to create release branches from current `viable/strict`. Perform following actions :
+* Perform a fresh clone of pytorch repo using
+```bash
+git clone git@github.com:pytorch/pytorch.git
+```
 
+* Change to pytorch repo root and run following commands:
```

Comment:
Please capitalize PyTorch, also, I'm not sure what `Change to pytorch repo root` means, perhaps it better be rephrased as follows
```suggestion
* Execute following command from PyTorch repository root folder:
```